tests/console/btrfsmaintenance.pm

```diff
@@ -41,45 +43,80 @@ sub btrfs_service_unavailable {
     }
 }
 
+sub btrfs_await_scrub {
+    if (is_sle("<15-SP4")) {
+        script_retry('btrfs scrub status / | grep -e "scrub started .* and finished after"', retry => 20, delay => 30);
```

Comment:
One minor comment,

Why do we need retry many times with short `delay` time for these functions? 

Does it make sense to reduce the `retry` value and increase the "delay` time?

Reply:
We could, but a delay of 30 seconds is rather generous already. If we increase this further then we also block the openQA workers unnecessarily long.

Also in the test runs this `script_retry` typically only takes 1-2 iterations to complete.

So, I would argue increasing this delay here does not bring us much more.